src/brasil/gov/vlibrasnews/subscribers.py

```diff
@@ -126,6 +129,10 @@ def create_translation(context, event):
     :rtype: bool
     :raises: VLibrasNewsError
     """
+    # Unpublish item
+    if event.action != 'publish':
+        return delete_translation(context, event)
```

Comment:
I think is a bad idea to call the subscriber this way; we should take the delete functionality out of it.